template/.eslintrc.js

```diff
@@ -19,6 +20,8 @@ module.exports = {
     {{#if_eq lintConfig "standard"}}
     // allow paren-less arrow functions
     'arrow-parens': 0,
+    // allow async-await
+    'generator-star-spacing': 0,
```

Comment:
Regarding this line, please refer to the following links:
- https://github.com/babel/babel-eslint/issues/316
- https://github.com/eslint/eslint/issues/6274